### PR TITLE
[Samples] Python: added support of onnx models

### PR DESCRIPTION
**Pull request by mryzhov**
- updated suitable samples
- updated documentation
- changed the name of hello_classification.py sample to align it with c/c++ samples)

### CODE REVIEW

**Comment by ilyachur** [comment]
As I see we have some code from this PR: https://github.com/openvinotoolkit/openvino/pull/2747 Can we remove this code from this PR?

**Comment by mryzhov** [comment]
> As I see we have some code from this PR: #2747 Can we remove this code from this PR?

yes, this is a branching issue, will fix that, thanks for pointing to that

File: inference-engine/ie_bridges/python/sample/classification_sample_async/classification_sample_async.py [hunk 1]
```diff
@@ -96,17 +96,23 @@ def build_argparser():
...
-    log.info("Loading network files:\n\t{}\n\t{}".format(model_xml, model_bin))
-    net = ie.read_network(model=model_xml, weights=model_bin)
+
+    # Read a model in OpenVINO Intermediate Representation (.xml and .bin files) or ONNX (.onnx file) format
+    model = args.model
+    model_bin = None
+    model_name, model_ext = os.path.splitext(model)
+    log.info(f"Loading network files:\n\t{model}")
+    if model_ext == ".xml":
+        # Read.bin weights for IR format only
```
Comment (generalova-kate) [inline inference-engine/ie_bridges/python/sample/classification_sample_async/classification_sample_async.py:112]: `# Read.bin weights for IR format only`
Add space between `Read` and `.bin`

File: inference-engine/ie_bridges/python/sample/classification_sample_async/README.md [hunk 2]
```diff
@@ -62,7 +62,8 @@ Running the application with the empty list of options yields the usage message
 To run the sample, you can use AlexNet and GoogLeNet or other image classification models. You can download the pre-trained models with the OpenVINO [Model Downloader](@ref omz_tools_downloader_README) or from [https://download.01.org/opencv/](https://download.01.org/opencv/).
 
 > **NOTE**: Before running the sample with a trained model, make sure the model is converted to the Inference Engine format (\*.xml + \*.bin) using the [Model Optimizer tool](../../../../../docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md).
-
+> 
+> The sample accepts models in ONNX format (.onnx) that do not require preprocessing.
```
Comment (generalova-kate) [inline inference-engine/ie_bridges/python/sample/classification_sample_async/README.md:66]: Change `do` to `does` in `that does not require preprocessing.`

Comment (mryzhov) [reply inference-engine/ie_bridges/python/sample/classification_sample_async/README.md:66]: model**s** do not require

File: inference-engine/ie_bridges/python/sample/classification_sample_async/classification_sample_async.py (hunk 1 above)
Comment (mryzhov) [reply inference-engine/ie_bridges/python/sample/classification_sample_async/classification_sample_async.py]: fixed

**Review by akuporos — APPROVED** [review]

File: inference-engine/ie_bridges/python/sample/classification_sample_async/classification_sample_async.py [hunk 3]
```diff
@@ -96,17 +96,23 @@ def build_argparser():
...
-    # Read IR
-    log.info("Loading network files:\n\t{}\n\t{}".format(model_xml, model_bin))
-    net = ie.read_network(model=model_xml, weights=model_bin)
+
+    # Read a model in OpenVINO Intermediate Representation (.xml and .bin files) or ONNX (.onnx file) format
+    model = args.model
+    model_bin = None
+    model_name, model_ext = os.path.splitext(model)
+    log.info(f"Loading network files:\n\t{model}")
+    if model_ext == ".xml":
```
Comment (ilyachur) [inline inference-engine/ie_bridges/python/sample/classification_sample_async/classification_sample_async.py:111]: If bin and xml files have the same name, you can pass only xml path to read_network.

Comment (mryzhov) [reply inference-engine/ie_bridges/python/sample/classification_sample_async/classification_sample_async.py:111]: Unfortunately no ( bridge function read_network throws an error for IR model without .bin file.
This function should be changed as part of another PR.

**Pull request merged**